src/hotspot/os_cpu/windows_x86/orderAccess_windows_x86.hpp

```diff
@@ -28,6 +28,7 @@
 // Included in orderAccess.hpp header file.
 
 #include <intrin.h>
+#include <immintrin.h>
```

Comment:
With Visual Studio this is already included by intrin.h

Reply:
ok. will remove it